packages/number-field/test/number-field.test.ts

```diff
@@ -147,6 +147,31 @@ describe('NumberField', () => {
             expect(el.focusElement.value).to.equal('-2.4');
         });
     });
+    describe('it handles values greater than 1000', () => {
+        it('correctly handles values greater than 1000 with step=1', async () => {
```

Comment:
Can we bump this test up into the `describe('Step', () => {` block so we don't need the new `describe`, please.